topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,653 @@
+
+In this section, we will run a sub-workflow that can run in parallel to **Taxonomy Profiling**, **Gene based pathogenic identification** and **Pathogen Tracking among all samples** sub-workflows, so feel free to run it in the time you are waiting for the other sub-workflows to finish running. 
+
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Import a reference genome FASTA file via link from [Zenodo]({{ page.zenodo_link }}) or Galaxy shared data libraries to both histories; `Barcode10` and `Barcode11`:
+>
+>    - `Salmonella_Ref_genome.fna.gz`: For both histories; `Barcode10` and `Barcode11`
+>
+>    ```text
+>    {{ page.zenodo_link }}/files/Salmonella_Ref_genome.fna.gz
+>    ```
+>    - From the shared data libraries in Galaxy, datasets can be also found under __Shared Data__, __Data libraries__, then __Biolytix__
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+{: .hands_on}
+
+> <hands-on-title>SNP based Pathogenic Identification</hands-on-title>
+>
+> 1. **Import the workflow** into Galaxy
+>    - Copy the URL (e.g. via right-click) of [this workflow]({{ site.baseurl }}{{ page.dir }}workflows/nanopore_snp_based_pathogenetic_identification.ga) or download it to your computer.
+>    - Import the workflow into Galaxy
+>
+>    {% snippet faqs/galaxy/workflows_import.md %}
+>
+> 2. Run **Workflow 4: Nanopore Datasets - SNP based Pathogenic Identification** {% icon workflow %} using the following parameters:
+>    - *"Send results to a new history"*: `No`
+>    - {% icon param-file %} *"Nanopore Preprocessed reads"*: `Nanopore processed sequenced reads` the output from **Filter Sequence By ID** from the **Pre-Processing** sub-workflow
+>    - {% icon param-file %} *"Reference Genome of Tested Strain/Pathogen"*: `Salmonella_Ref_genome.fna.gz` file you have just uploaded to your history 
+>
+>    {% snippet faqs/galaxy/workflows_run.md %}
+>
+> 3. Run the same sub-workflow for the second history
+>
+>    {% snippet faqs/galaxy/histories_switching.md %}
+>
+{: .hands_on}
+
+
+## Variant Calling or SNP Calling
+
+To identify variants, the following tools are used:
+
+1. First we map to a reference genome of the speices of the pathogen you want to test your samples against, we used **Minimap2** since our datasets are Nanopore. 
+2. Then we indetify Variants and Single nucleotide variants using [Clair3](https://hpc.nih.gov/apps/Clair3.html), which is designed specifically for Nanopore datasets giving better results than other variant calling tools as well as its new and up-to-date. [__Medaka consensus tool__ and __medaka variant tool__](https://github.com/nanoporetech/medaka) can be also used instead of **Clair3**, they give similar results but they are much slower then **Clair3** and with much fewer tools' options. So we recommend **Clair3** that we used in this sub-workflow.
```

Comment:
```suggestion
2. Then we indetify variants and single nucleotide variants using [Clair3](https://hpc.nih.gov/apps/Clair3.html), which is designed specifically for Nanopore datasets giving better results than other variant calling tools as well as its new and up-to-date. [__Medaka consensus tool__ and __medaka variant tool__](https://github.com/nanoporetech/medaka) can be also used instead of **Clair3**, they give similar results but they are much slower then **Clair3** and with much fewer tools' options. So we recommend **Clair3** that we used in this sub-workflow.
```